app/models/concerns/job_posting_searchable.rb

```diff
@@ -25,7 +25,10 @@ module JobPostingSearchable
       attribute(:required_skills) { required_skills.body&.to_plain_text }
       attribute(:preferred_skills) { preferred_skills.body&.to_plain_text }
 
-      searchableAttributes %w[unordered(title) organization_display_name unordered(plain_description)]
+      searchableAttributes %w[
+        unordered(title) organization_display_name unordered(plain_description) identification_number
```

Comment:
Since we had some search requests with an identification number, I added it to the searchable attributes